src/flash-source-buffer.js

```diff
@@ -169,6 +169,12 @@ export default class FlashSourceBuffer extends videojs.EventTarget {
     };
 
     chunkInData();
+
+    // fire loaded metadata on first segment append
+    if (!this.firstAppendHappened) {
```

Comment:
From https://www.w3.org/TR/html51/semantics-embedded-content.html#eventdef-media-loadedmetadata , do we have enough information to fire loadedmetadata at appendBuffer, or do we have to wait until the buffer is finished appending?